test/test_target.py

```diff
@@ -0,0 +1,58 @@
+from unittest.case import TestCase
+try:
+    from unittest.mock import MagicMock
```

Comment:
just add `mock` to requirements and remove this try/except, leave the `from mock import MagicMock`